### PR TITLE
avian: delete

### PR DESCRIPTION
**Pull request by Bo98**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by Bo98** [comment]
Found some security patches applied upstream. Amended this PR accordingly.

* CVE-2020-17360
* CVE-2020-17361
* CVE-2020-28371

File: Formula/avian.rb [hunk 1]
```diff
@@ -22,7 +23,25 @@ class Avian < Formula
 
   uses_from_macos "zlib"
 
+  # Fixes:
+  # - CVE-2020-17360
+  # - CVE-2020-17361
+  # - CVE-2020-28371
```
Comment (SMillerDev) [inline Formula/avian.rb:29]: Will these be in the next release?

Comment (Bo98) [reply Formula/avian.rb:29]: There will not be a next release. This formula is deprecated as unmaintained.

Maybe we should just disable it, as I don't know why it crashes on Catalina and Big Sur (probably some obscure compiler change).

Comment (SMillerDev) [reply Formula/avian.rb:29]: Unmaintained, can't build and has security issues. I think it ticks all boxes to be disabled.

**Comment by jonchang** [comment]
> ==> Analytics
> install: 0 (30 days), 1 (90 days), 36 (365 days)
> install-on-request: 0 (30 days), 1 (90 days), 35 (365 days)
> build-error: 0 (30 days)

I think we can probably just delete this without any complaint.

**Comment by jonchang** [comment]
> ==> Analytics
> install: 0 (30 days), 1 (90 days), 36 (365 days)
> install-on-request: 0 (30 days), 1 (90 days), 35 (365 days)
> build-error: 0 (30 days)

I think we can probably just delete this without any complaint.

**Comment by Bo98** [comment]
Fine with me.

**Review by jonchang — APPROVED** [review]

**Pull request merged**